### ISSUE TITLE
about the dataset

### ISSUE THREAD

**Issue by HalbertCH** [MingtaoGuo/BigGAN-tensorflow#2]
Hello, I saw you used the "imagenet64.mat" file as your dataset, and I think the file is much more convenient to use than those original imagenet images. Now I want to train my model one the 128*128 imagenet dataset, do you know if there exists an "imagenet128.mat" file? If exists, do you know where to download the file? Thank you very much!

**Comment by MingtaoGuo** [comment]
If you want, i will help you to generate imagenet128.mat, but it will cost more memory than imagenet64.mat.
My E-mail: gmt798714378@hotmail.com

**Comment by HalbertCH** [comment]
I don't need it anymore, but thank you all the same！Besides, I have another question, that is, I saw you used the learning rate of 1e-4 in D and 4e-4 in G in your code, while in the papers of BigGAN and SAGAN the learning rate of D is larger, which is opposite to your setting. May I ask why you set the learning rate in such a differnt way?
